features/container/sentinel/src/main/filtered-resources/etc/org.apache.karaf.features.cfg

```diff
@@ -54,6 +54,10 @@ featuresBoot = \
     kar/${karafVersion}, \
     scv-jceks-impl/${project.version}, \
     scv-shell/${project.version}, \
+    owasp-encoder, \
```

Comment:
If we need these features for `opennms-health-rest-service` to work, we need them to be part that feature instead of adding them here.